### ISSUE TITLE
Show the selected char when drawing

### ISSUE THREAD

**Issue by Esshahn** [nurpax/petski#7]
When drawing, only the rectangle border is shown.

Instead, the selected character should be displayed, so that one can immediately see
- what character you're drawing with
- how it would like if you click (before you click)

**Comment by nurpax** [comment]
Agreed. This is how I had it earlier but it got dropped in some refactoring but I intended to add it back.

**Comment by nurpax** [comment]
Done for showing current char.  But need colorize support too.
